Source/arcadia/internal/components.clj

```diff
@@ -17,6 +17,14 @@
   (->> (range \a \z)
        (map (comp str char))))
 
+(def overrides
+  "Which methods of ArcadiaBehaviour should be overriden/hidden."
+  '{Awake override})
+
+(def call-base
+  "Which overriden methods should call the base method."
+  '{Awake true})
+
```

Comment:
Is there ever a situation when you would *not* call the base method in an overridden method? My feeling is that we could just use the `overrides` map and not have to keep two maps in sync.

Realistically though this is the only method that will need this, so its not a terribly big deal.